libs/viewer/src/SimpleViewer.cpp

```diff
@@ -985,17 +988,12 @@ void SimpleViewer::updateUserInterface() {
             ImGui::Unindent();
         }
 
-        if (!mMorphWeights.empty() && ImGui::CollapsingHeader("Morphing")) {
```

Comment:
Do the sliders move if an animation is underway?  Should they be disabled while an animation is underway?  How do we guarantee that the weights sum to 1.0?

Reply:
I think sliders should move when the animation is underway :-) That goes well with the ImGUI realtime everything way.

I think we don't need to enforce in the UI that all weights sum to 1 -- it's good for testing. However, we could have a text box that displays the sum. @daemyung can you add that?

Reply:
Re: summing to 1.0, doh, I was thinking of skinning!

@daemyung this is a really cool feature, but I want to avoid getting bug reports like: "the sliders don't work" because users try them on models that have animation that affect morph weights, in which case the sliders won't do anything.

Note that when `mCurrentAnimation == 0`  in `SimpleViewer`, that means animation is disabled, so we could perhaps gray out the sliders when that's false.

For the "sliders move on their own" idea that Mathias likes, maybe we could expose more of the Animator internals, but I'm reluctant to do that.  I would like to keep the simplicity of the Animator API (gltfio is just a loader).  Is there any another way we could do it?  For example maybe it wouldn't be so bad to track the values in the app.

Reply:
@prideout I think to gray out the sliders is the best solution now. I will do it that on the different PR.